tests/python/mkl/test_subgraph.py

```diff
@@ -727,6 +731,7 @@ def test_pos_conv_add():
     check_fusion(net, data_shape, attrs)
 
 @with_seed()
+@unittest.skip('skip for MKL-DNN 1.0 integration: https://github.com/apache/incubator-mxnet/projects/16')
```

Comment:
These will be resolved before merge, right?

Reply:
Sure. Will merge https://github.com/apache/incubator-mxnet/pull/16565 into the branch.

Reply:
Merged:)